torch/_decomp/__init__.py

```diff
@@ -53,7 +53,18 @@ def add_op_to_table(aten_op):
                 if op_overload in registry:
                     raise RuntimeError(f"duplicate registrations for {op_overload}")
```

Comment:
could we change the above to `torch.ops.OpOverloadPacket` and `torch.ops.OpOverload`

Reply:
but that spelling doesn't work? 

```
>>> torch._ops.OpOverloadPacket
<class 'torch._ops.OpOverloadPacket'>
>>> torch.ops.OpOverloadPacket
<module 'torch.ops.OpOverloadPacket' from 'torch.ops'>
```

Reply:
ugh okay. I am not sure what to do but it seems like these classes should be accessible from a namespace not beginning with an underscore since they are pretty much a public API at this point.

Reply:
Another fix is to put them in another module, not torch._ops